browser/ui/views/side_panel/brave_side_panel_coordinator.cc

```diff
@@ -146,3 +149,20 @@ void BraveSidePanelCoordinator::UpdateToolbarButtonHighlight(
                            : IDS_TOOLTIP_SIDEBAR_SHOW));
   }
 }
+
+void BraveSidePanelCoordinator::PopulateSidePanel(
+    bool supress_animations,
+    SidePanelEntry* entry,
+    std::optional<std::unique_ptr<views::View>> content_view) {
+  actions::ActionItem* const action_item = GetActionItem(entry->key());
```

Comment:
maybe also `CHECK(entry)` just in case?

Reply:
If we put `CHECK(entry)` here, below code run when `entry` is nullptr?

Reply:
Ah, confused with `action_item`. Added `CHECK(entry);`